appveyor.yml

```diff
@@ -17,6 +17,7 @@ install:
   - go version
   - go env
   - python --version
+  - go get -u golang.org/x/crypto/argon2
```

Comment:
this is not needed if you have vendorized it already what is the error that you see? @poornas

Reply:
removed this.